src/demo/DemoTestLoanService.java

```diff
@@ -285,9 +285,10 @@ public static void testChangeInterestRateForActiveRevolvingCreditLoans() throws
 			calendar.add(Calendar.MONTH, 1);
 			jsonTransactionRequest.setDate(calendar.getTime());
 			jsonTransactionRequest.setRate(new BigDecimal(10));
-			jsonTransactionRequest.setNotes("some notes");
+			jsonTransactionRequest
+					.setNotes("Interest rate changet through API to be " + jsonTransactionRequest.getRate() + "%");
 
-			LoanTransaction loanTransaction = loanService.executeJSONTransactionRequest(NEW_LOAN_ACCOUNT_ID,
+			LoanTransaction loanTransaction = loanService.postInterestRateChange(NEW_LOAN_ACCOUNT_ID,
 					LoanTransactionType.INTEREST_RATE_CHANGED, jsonTransactionRequest);
 
 			System.out.println("The interest rate was edited for the loan account " + NEW_LOAN_ACCOUNT_ID
```

Comment:
Could you please log the transaction ID here, instead of the entire transaction?